clients/cobol-lsp-vscode-extension/src/__tests__/CopybookResolveURITest.ts

```diff
@@ -110,3 +116,51 @@ describe("With allowed input parameters, the list of URI that represent copybook
         expect(buildResultArrayFrom(["HLQ.DATASET1.DATASET2"], "PRF")).toBe(1);
     });
 });
+
+describe("Prioritize search criteria for copybooks test suite", () => {
+    function provideMockValueForLocalAndDSN(localPath: string, dsnPath: string) {
+        vscode.workspace.getConfiguration = jest.fn().mockReturnValueOnce({
+            get: jest.fn().mockReturnValueOnce([localPath]),
+        }).mockReturnValueOnce({
+            get: jest.fn().mockReturnValueOnce([dsnPath]),
+        });
+    }
+
+    const profileService: ProfileService = new ProfileService(undefined);
+    const copybookDownloadService: CopybookDownloadService = new CopybookDownloadService(undefined, undefined, undefined, undefined);
+    const copybookURI: CopybookURI = new CopybookURI(profileService, copybookDownloadService);
+    profileService.getProfile = jest.fn().mockReturnValue("PRF");
+    copybookDownloadService.downloadCopybook = jest.fn().mockReturnValue("CPY");
+
+    const spySearchInWorkspace = jest.spyOn(CopybookURI, "searchInWorkspace");
+    test("With only a local folder defined in the settings.json, the search is applied locally", async () => {
+        vscode.workspace.getConfiguration = jest.fn().mockReturnValue({
+            get: jest.fn().mockReturnValue([CPY_FOLDER_NAME]),
+        });
+
+        const uri: string = await copybookURI.resolveCopybookURI(copybookName, "PRGNAME");
+        expect(uri.includes(CPY_FOLDER_NAME)).toBe(true);
```

Comment:
You could use this: `expect(uri).toMatch(CPY_FOLDER_NAME);`

Reply:
OK